api/docs/docs/updates.md

```diff
@@ -7,6 +7,27 @@ description: Updates to the Covid Act Now API.
 
 Updates to the API will be reflected here.
 
+### CDC Community Level data now available 
+_Added on 2022-04-05_
+
+The CDC Community Level metric as well as the subcomponents of the CDC Community Level metric are now available within the Covid Act Now API. 
+
+[Learn more about how the CDC Community Level is measured](https://covidactnow.org/covid-risk-levels-metrics). 
```

Comment:
The link will need to be updated to point directly to CDC Community Level section once it is live